### ISSUE TITLE
Close assistant

### ISSUE THREAD

**Issue by adamsoutar** [zed-industries/community#2048]
### Check for existing issues

- [X] Completed

### Describe the feature

This seems like such a silly ticket to write - hopefully I'm just being dumb and I've missed something 😅

Is it possible to close Assistant?

I hit the magic wand icon because I was curious about what it was, and that opened the panel asking for an API key.

I'm not sure what I'm looking for, but hitting the wand again doesn't get rid of it, pressing `Cmd+W` doesn't get rid of it, `Esc` doesn't get rid of it, double-clicking the separator bar doesn't get rid of it, putting in an actual key and then looking for buttons, there's still no way to get rid of it.

The best I can do is drag it across to like 10px wide - it has a minimum size. Is there a way to close it?

**Comment by JosephTLyons** [comment]
Hey @adamsoutar, what you're looking for is the button used to close the associated dock that is housing the assistant.  In my case, its on the left side, but it might be on th right side for you.  It looks like a `Z`.

<img width="640" alt="SCR-20230917-ttad" src="https://github.com/zed-industries/community/assets/19867440/03471f5e-bd4b-4d5f-9271-acd8f4f6ca1f">

**Comment by JosephTLyons** [comment]
Something to note is that assistant isn't tied directly to any specific dock, but whatever dock you decide to put it in (right or left in this case).

The model might be easier to understand when you think about the project panel, which works the same way.  You put it in your preferred dock (right or left) and the the project panel button opens and closes that dock.  Same thing happens for assistant, the terminal, and the collaboration panel, which all have buttons in the status bar that trigger their dock to open.

I think there are still ways we can help make things more obvious to the user, so I've tagged this as a discoverability issue.

**Comment by tkgalk** [comment]
Yeah, this has to be fixed, I had to do a Google search to find a GitHub issue to close a panel. Definitely needs some UX work. 🙈
